js/whatweekisit.js

```diff
@@ -20,7 +20,13 @@ function storageAvailable(type) {
 function getWeek() {
     nowMoment = moment();
     for (var i = 0; i < weeks.length; i++) {
-        if (nowMoment.isBetween(weeks[i].beginning, weeks[i].end, "day")) {
+        var beginning = weeks[i].beginning;
+        var end = weeks[i].end;
+        //Note: isBetween() is exclusive. This could be done with less || statements,
+        //but I left the isBetween() call in case it is one day inclusive. -VB 1/15/16
+        var withinWeek = nowMoment.isBetween(beginning, end, "day");
+        withinWeek = withinWeek || nowMoment.isSame(beginning) || nowMoment.isSame(end);
```

Comment:
I created the `withinWeek` variable because I wasn't sure how picky you were with code style, and it was going to be a really long line if I didn't make a variable/split it into two lines. Feel free to recombine if you don't care.